packages/webapp/src/webview/store/middleware.ts

```diff
@@ -35,3 +41,19 @@ export const postMessageMiddleware: Middleware = (store: MiddlewareAPI) => {
 export const loggerMiddleware = createLogger({
     duration: true
 });
+
+const allowedTelemetryActions = new Set([LOG_TELEMETRY_EVENT]);
```

Comment:
The `allowedTelemetryActions` is the list of redux actions, that are allowed to be logged and therefore should be send via `postMessage()` to ide-extensions. In this case this might be:

```suggestion
const allowedTelemetryActions = new Set([KHUB_OPEN_TUTORIAL, KHUB_OPEN_BLOGS]);
```